### ISSUE TITLE
Flexible settings

### ISSUE THREAD

**Issue by TedAlter** [SensorApps/Sensors2OSC#46]
It would be good if I could set sending of  (for example) **/georotationvector** arguments as different lines (as /georotationvectorrotation , /georotationvectorpitch ...). (if this function realizated as "Send OSC values as bundle" it don't work)

**Comment by residuum** [comment]
"Send OSC values as bundle" is a workaround for OSC receivers, that cannot parse OSC messages with multiple values, but only OSC bundles. For details see https://opensoundcontrol.stanford.edu/spec-1_0.html

Prior to version 0.3.0, all messages where single values with all parameters as single values, so you could select individual values to send. As most applications use most values as a whole, sensorwise the load on the device is the same, and sending more values over the network, this leads to more delay of the data from sender to receiver, especially when more values are selected, or several senders are used for the same receivers over the same network, then there is more things to configure, explain in help, and more hassle to debug, I have decided to leave the separation of values to the receiver end.

TL;DR: I will not implement it, but I am open to pull requests which includes this behaviour for all values and includes possible settings in the help.
